scalalib/src/publish/VersionScheme.scala

```diff
@@ -0,0 +1,12 @@
+package mill.scalalib.publish
+
+sealed abstract class VersionScheme(val value: Option[String])
+
+object VersionScheme {
+  case object NoScheme extends VersionScheme(None)
+  case object Always extends VersionScheme(Some("always"))
```

Comment:
This one is not a proper version scheme, it may only be used by users of libraries as an escape hatch to silence an eviction error.

Reply:
This needs to be addressed.